run-in-retro-term.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# cool-retro-term support for tetris.sh
+# Author: Rojen Zaman <rojen@riseup.net>
+# License: WTFPL
+
+set -u                                                    # return error if string null
+stdout="/dev/null"                                        # default stdout, optional verbose mode is /dev/stdout
```

Comment:
Do you think there is chance this will be redefined? I'm asking because there is not much space saving with this substitution, but from my perspective having explicit /dev/null helps script readability.

Reply:
Ah, sorry, missed that this is used in the verbose mode, please disregard my comment above.